src/Redux/configureStore.js

```diff
@@ -0,0 +1,20 @@
+import { applyMiddleware, legacy_createStore as createStore } from 'redux';
+import thunk from 'redux-thunk';
+import reducer from './covidData/countriesCovidData';
+
+const store = createStore(reducer, applyMiddleware(thunk));
+
+export default store;
+// import { applyMiddleware, combineReducers, createStore } from 'redux';
+// import thunk from 'redux-thunk';
+// import rocketReducer from './rockets/rockets';
+// import missionsReducer from './missions/missions';
+
+// const rootReducer = combineReducers({
+//   rocket: rocketReducer,
+//   missions: missionsReducer,
+// });
+
+// const store = createStore(rootReducer, applyMiddleware(thunk));
+
+// export default store;
```

Comment:
- [ ] OPTIONAL
There are commented code blocks found in your code. Kindly endeavor to remove **all** commented code blocks from your code before pushing it to the repository so as to keep your code well organized.